pkg/volume/util.go

```diff
@@ -410,3 +410,27 @@ func JoinMountOptions(userOptions []string, systemOptions []string) []string {
 	}
 	return allMountOptions.UnsortedList()
 }
+
+// ZonesToSet converts a string containing a comma separated list of zones to set
+func ZonesToSet(zonesString string) (sets.String, error) {
+	zonesSlice := strings.Split(zonesString, ",")
+	zonesSet := make(sets.String)
+	for _, zone := range zonesSlice {
+		trimmedZone := strings.TrimSpace(zone)
```

Comment:
It is necessary to convert the input string with list of zones to a set because the [func ChooseZoneForVolume ](https://github.com/kubernetes/kubernetes/blob/master/pkg/volume/util.go#L295) requires a set as one of the input parameters. As the `strings.Fields` converts a string to a slice a helper function will be needed anyway. However, the line `trimmedZone := strings.TrimSpace(zone)` could be deleted if `strings.Fields` is used.
IMHO, comma separated list feels better than a space separated list. That's why I prefer to keep it as it is.